pyvrp/cpp/ProblemData.h

```diff
@@ -346,9 +347,12 @@ class ProblemData
      * fixed_cost
      *     Fixed cost of using a vehicle of this type. Default 0.
      * tw_early
-     *     Start of the vehicle type's shift. Default 0.
+     *     Earliest start of the vehicle type's shift. Default 0.
+     * start_late
+     *     Latest start of the vehicle type's shift. Default None.
      * tw_late
-     *     End of the vehicle type's shift. Unconstrained if not provided.
+     *     Latest finish of the vehicle type's shift. Unconstrained if not
+     *     provided.
```

Comment:
Can we revert the `tw_early` and `tw_late` docstrings? In general, for things that are not directly relevant to the PR, I try not to touch them to reduce the diff size as much as possible.

Reply:
I do think that this change in the docstrings makes it more clear what the distinction is between `tw_early/tw_late/start_late` and how it works. But I will revert the change as you wish.